built-in-references/Kubernetes/container-disallowed-capabilities/template.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sazuredisallowedcapabilities
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sAzureDisallowedCapabilities
+      validation:
+        # Schema for the `parameters` field
+        openAPIV3Schema:
+          properties:
+            allowedCapabilities:
+              type: array
+              items:
+                type: string
+            requiredDropCapabilities:
```

Comment:
Was this policy tested? 
I see the constraint.yaml supplies parameter disallowedCapabilities and template.yaml declares allowedCapabilities, requiredDropCapabilities as parameters.

Reply:
Discussed offline - updated the policy